src/content/docs/apm/apm-ui-pages/events/change-tracking/deployments-page-view-impact-your-app-users.mdx

```diff
@@ -0,0 +1,189 @@
+    />
+    <Callout variant="tip">
+      Note that you may need to adjust the selected time range on this page to see deployments. Also, by default the table will only lazy load up to 2,000 deployments. Keep this in mind as you're filtering, searching, and adjusting your time range.
+    </Callout>
+* **New Relic Alerts & Detection:** If your deployment is related to an issue, you'll see it listed on the Deployments table of the **Root Cause Analysis** section of the **Issues** page.
+    <img
+      title="A screenshot showing the root cause analysis page in the UI"
+      alt="A screenshot showing the root cause analysis page in the user interface"
+      src={trackingRootCauseAnalysis}
+    />
+* **Activity stream newsfeed component:** You can see recorded deployment events in the activity feed on the right collapsible panel on various pages across the New Relic UI.
+    <img
+      title="A screenshot showing the activity stream in the right pane of the UI"
+      alt="A screenshot showing the activity stream in the right pane of the UI"
+      src={trackingActivityStreamExample}
+    />
+
+## Analyze the impact of your deployment [#impact]
+
+How have your deployments affected your sytems? We offer a number of ways to analyze the effects of your deployments.
+
+### Deployment details page [#deployment-details]
+
+Clicking on a deployment virtually anywhere in New Relic—on a chart, on the **Issues** page, and so on—will open a **Deployment details** page containing the data you sent when you recorded the deployment plus a wealth of information about its impact.
+
+You'll get various insights into how this deployment may have have affected this entity's health, performance, and quality. These insights are based on errors, key metrics and log attribute trends, anomalies, issues, and more. This page also features powerful scoping and comparison tools to help you see and understand even more context about ongoing changes. For example, you can see how this deployment compares to the one before.
+
+    <img
+      title="A screenshot showing and example of the deployment details page"
+      alt="A screenshot showing and example of the deployment details page"
+      src={trackingDeploymentDetails}
+    />
+
+Here are some helpful insights you can get by recording deployments and leveraging other New Relic features:
+
+* Calculated net impacts of the change on key entity signals, such as throughput, error rate, transaction time, and more. You can see the net impact on a signal above each time series chart (typically shown as a percentage). Note that these net impacts are usually calculated by comparing averages or some other function of the signal before and after the deployment (hover over net impact figures to see before/after averages, etc.).
+
+You can adjust the before/after period used in these calculations and displayed on time series charts via the timepicker at the top right corner of the deployment details page.
```

Comment:
Shouldn't be this paragraph and the tip note at the same level in the starting paragraph "Calculated net..."?

Reply:
HI @rhetoric101, I still see the indentation of this paragraph and of the tip note outside the bulleted paragraph.

Reply:
@nbaenam I just tried again to fix this. Let me know if this looks OK!

Reply:
Sure! I'll do it as soon as the build finishes!